automation/common/versions.sh

```diff
@@ -59,5 +60,5 @@ KUBEVIRT_VERSION=$(latest_version "kubevirt" "kubevirt")
 # Latest released CDI version
 CDI_VERSION=$(latest_version "kubevirt" "containerized-data-importer")
 
-# Latest released Tekton version
-TEKTON_VERSION=$(latest_version "tektoncd" "operator")
+# Using LTS tekton version v0.67
+TEKTON_VERSION=$(latest_patch_version "tektoncd" "operator" "v0.67")
```

Comment:
Are you sure that `v0.67` exists? I don't see it and only see `v0.67.0` [1].

[1] https://api.github.com/repos/tektoncd/operator/releases